### ISSUE TITLE
Error: 'clean_negidx' at (1) is not a member of the 'psb_base_sparse_mat' structure

### ISSUE THREAD

**Issue by sagitter** [sfilippone/mld2p4-2#4]
Hi all.

Compiling of `mld2p4-2.2.1-1` is failing on _Fedora 33_ with this error:

```
make[3]: Entering directory '/builddir/build/BUILD/mld2p4-2-2.2.1-1/mld2p4-2-2.2.1-1/mlprec/impl/aggregator'
gfortran -O2 -g -pipe -Wall  -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fPIC -I/usr/include/openblas  -I../.. -I../../../modules  -I../../../include -I/usr/lib64/gfortran/modules/psblas3 -I/usr/include/psblas3 -c mld_s_spmm_bld_inner.f90 -o mld_s_spmm_bld_inner.o
mld_s_spmm_bld_inner.f90:161:27:

  161 |   call tmpcoo%clean_negidx(info)
      |                           1
Error: 'clean_negidx' at (1) is not a member of the 'psb_base_sparse_mat' structure
mld_s_spmm_bld_inner.f90:87:12:

   87 |   naggr   = nlaggr(me+1)
      |            1
Warning: Possible change of value in conversion from INTEGER(8) to INTEGER(4) at (1) [-Wconversion]
mld_s_spmm_bld_inner.f90:88:12:

   88 |   ntaggr  = sum(nlaggr)
      |            1
Warning: Possible change of value in conversion from INTEGER(8) to INTEGER(4) at (1) [-Wconversion]
mld_s_spmm_bld_inner.f90:89:12:

   89 |   naggrm1 = sum(nlaggr(1:me))
      |            1
Warning: Possible change of value in conversion from INTEGER(8) to INTEGER(4) at (1) [-Wconversion]
mld_s_spmm_bld_inner.f90:90:12:

   90 |   naggrp1 = sum(nlaggr(1:me+1))
      |            1
Warning: Possible change of value in conversion from INTEGER(8) to INTEGER(4) at (1) [-Wconversion]
make[3]: *** [../../../Make.inc:111: mld_s_spmm_bld_inner.o] Error 1
```

Full build log (including dependencies list): https://download.copr.fedorainfracloud.org/results/sagitter/ForTesting/fedora-rawhide-x86_64/01329233-mld2p4/builder-live.log.gz

**Comment by sfilippone** [comment]
My bad, should be fixed with PSBLAS release Version 3.6.1-2
<https://github.com/sfilippone/psblas3/releases/tag/V3.6.1-2>
Let me know if it works

**Comment by sagitter** [comment]
> Let me know if it works

`mld2p4` is compiled fine against the release `3.6.1-2`of `psblas3`.
